### PR TITLE
Disable Windows builds and ATH to mitigate the ongoing INFRA issue

### PR DESCRIPTION
**Pull request by oleg-nenashev**
Some emergency measures to get the CI running. See https://groups.google.com/forum/#!topic/jenkinsci-dev/v9d-XosOp2s 

@jenkinsci/core

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -94,11 +97,12 @@ builds.ath = {
...
         }
     }
-}
+}*/
 
 builds.failFast = failFast
 parallel builds
-infra.maybePublishIncrementals()
+// TODO: Restore Incrementals once https://groups.google.com/forum/#!topic/jenkinsci-dev/v9d-XosOp2s is resolved
+// infra.maybePublishIncrementals()
```
Comment (jglick) [inline Jenkinsfile]: Fine as a hotfix, but actually what we should do _anyway_ (permanently) is switch this method to use an ACI agent.

**Review by jglick — APPROVED** [review]

File: Jenkinsfile (hunk 1 above)
Comment (oleg-nenashev) [reply Jenkinsfile]: Do not disagree, should be done in https://github.com/jenkins-infra/pipeline-library

Comment (jglick) [reply Jenkinsfile]: I will prepare a patch for it.

Comment (jglick) [reply Jenkinsfile]: https://github.com/jenkins-infra/pipeline-library/pull/107

**Review by jeffret-b — APPROVED** [review]
These changes make sense to get us moving forward.

**Comment by oleg-nenashev** [comment]
Should be OK according to https://ci.jenkins.io/blue/organizations/jenkins/Core%2Fjenkins/detail/jenkinsfile-windows-disable/1/pipeline/7 . Will merge once the build passes

File: Jenkinsfile (hunk 1 above)
Comment (jglick) [reply Jenkinsfile]: Merged, so I think this hunk could now be reverted.

**Review by jglick — APPROVED** [review]

**Pull request merged**